tests/functional/Ticket.php

```diff
@@ -5719,4 +5719,172 @@ public function testGetEntitiesForRequesters()
         $entities = $ticket->getEntitiesForRequesters(["_users_id_requester" => $user_id]);
         $this->array($entities)->isIdenticalTo([$entity2_id, $entity1_id]);
     }
+
+    public function testAssignToMe()
+    {
+        $this->login();
+        $entity_id = 0;
+
+        //create a ticket
+        $ticket = new \Ticket();
+        $ticket_id = $ticket->add([
+            'name'                  => __METHOD__,
+            'content'               => __METHOD__,
+            'entities_id'           => $entity_id,
+            '_skip_auto_assign'     => true,
+            '_users_id_requester'   => getItemByTypeName('User', 'normal', true),
+        ]);
+        $this->integer($ticket_id)->isGreaterThan(0);
+
+        $_SESSION['glpiset_followup_tech'] = 0;
+
+        //add a followup to the ticket without assigning to me
+        $fup = new \ITILFollowup();
+        $this->integer(
+            (int)$fup->add([
+                'itemtype'  => 'Ticket',
+                'items_id'  => $ticket_id,
+                'users_id'  => getItemByTypeName('User', 'tech', true),
```

Comment:
`Ticket::assignToMe()` will check for current connected user rights (see `canAssignToMe`), so you should login with `tech` user before adding a followup.

Thus, if current user is `tech`, it will be automatically defined as followup author.
```suggestion
```

Same remark for other tests.